atomium/includes/common.inc

```diff
@@ -387,7 +387,7 @@ function atomium_menu_get_active_breadcrumb() {
 
     // Don't show a link to the current page in the breadcrumb trail.
     $end = \end($active_trail);
-    if ($item['href'] == $end['href']) {
+    if ($item['href'] === $end['href']) {
```

Comment:
This change is safe if we are comparing two strings.
Both values are provided and manipulated by external code, so in theory could be anything.

Again, this should perhaps be a dedicated commit with a message that indicates potential behavior change.